src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.test.js

```diff
@@ -96,6 +96,9 @@ describe('MetaList', () => {
       expect(screen.queryByTestId('meta-list-handling-time-definition')).toHaveTextContent(/^Afhandeltermijn$/);
       expect(screen.queryByTestId('meta-list-handling-time-value')).toHaveTextContent(/^4 werkdagen$/);
 
+      expect(screen.queryByTestId('meta-list-process-time-definition')).toHaveTextContent(/^Doorlooptijd$/);
```

Comment:
This is correct, but only because the fixture does not have a `deadline` or `deadline_factor_3`. Can you add that to the incident fixture? Perhaps in a way that you'd get "3x buiten de afhandeltermijn" in order for tests not to break in the future